config/locales/pages/contact_ea.en.yml

```diff
@@ -0,0 +1,12 @@
+---
+en:
+  pages:
+    contact_ea:
+      heading:  Contact the Environment Agency
+      subheading: You chose ‘I don’t know’ for the organisation type
+      call_to_action: >
+        Please call our helpline to get advice about who might be the organisation or
+        person responsible for your activity.
+      contact_tel: 'Telephone: 03708 506 506'
+      contact_minicom: 'Minicom: 03702 422 549'
+      contact_times: Monday to Friday, 8am to 6pm
```

Comment:
Should this data be coming from (or may be overriden by) environment variables as these details are likely to change.

Also - should use double quotes to be consistent with rest of app.